web/themes/custom/usagov/usagov.theme

```diff
@@ -444,3 +444,65 @@ function usagov_theme_suggestions_alter(array &$suggestions, array $variables, $
     }
   }
 }
+
+function usagov_preprocess_breadcrumb(&$variables) {
+
+  $database = \Drupal::database();
+
+  if (strpos($_SERVER['REQUEST_URI'], 'agencies/') !== FALSE) {
```

Comment:
to test if the current page is the federal agency page, see `TaxonomyDatalayerBuilder::isFederalDirectoryIndex()`

```php
// get current node
$node = \Drupal::routeMatch()->getParameter('node');
// check if agency index is english page
if ($this->node->toUrl()->toString() === TaxonomyDatalayerBuilder::AGENCY_INDEX_URL_EN) {
```

Similar test for Spanish page, uses a different constant.